phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -722,14 +722,27 @@ public MetaDataMutationResult updateCache(PName origTenantId, String schemaName,
     // 1. table is a system table that does not have a ROW_TIMESTAMP column OR
     // 2. table was already resolved as of that timestamp OR
     // 3. table does not have a ROW_TIMESTAMP column and age is less then UPDATE_CACHE_FREQUENCY
+    // 3a. Get the effective UPDATE_CACHE_FREQUENCY for checking the age in the following precedence order:
+    // Table-level property > Connection-level property > Default value.
```

Comment:
Please create a follow-up JIRA for this documentation change, particularly about the precedence order. Maybe we can add a couple of lines here https://phoenix.apache.org/language/index.html#options

Reply:
Done. Created [PHOENIX-5653](https://issues.apache.org/jira/browse/PHOENIX-5653)